### PR TITLE
Migrate debugging support to java 11

### PR DESCRIPTION
**Pull request by NipunaRanasinghe**
## Purpose

This PR 
- Re-enables debugger build in jdk11 based ballerina distribution build.
- Updates debug adapter launcher scripts with java 11 debug agent args.
- Updates debug server implementation and debug adapter CLI command for java 11 support.
- Enables debugger variable and expression evaluation tests.
- Fixes https://github.com/ballerina-platform/ballerina-lang/issues/26226.

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [x] Checked Tooling Support (#<Issue Number>)
- [x] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [x] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/utils/DebugUtils.java [hunk 1]
```diff
@@ -9,9 +9,9 @@
  */
 public class DebugUtils {
 
-    private static final String DEBUG_ARGS_JAVA8 = "-agentlib:jdwp=transport=dt_socket,server=y,suspend=y";
```
Comment (praveennadarajah) [inline cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/utils/DebugUtils.java:12]: Seems like debugging will only support in Java 11. Don't we have to support both Java 8 and Java 11?

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/utils/DebugUtils.java [hunk 2]
```diff
@@ -9,9 +9,9 @@
...
+    private static final String DEBUG_ARGS_JAVA11 = "-agentlib:jdwp=transport=dt_socket,server=y,suspend=y";
     private static final String JAVA_VERSION_PROP = "java.version";
-    private static final String COMPATIBLE_JRE_VERSION = "1.8";
+    private static final String COMPATIBLE_JRE_VERSION = "11";
```
Comment (praveennadarajah) [inline cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/utils/DebugUtils.java:14]: Seems like debugging will only support in Java 11. Don't we have to support both Java 8 and Java 11?

Comment (NipunaRanasinghe) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/utils/DebugUtils.java:14]: @praveennadarajah If I'm not mistaken, our ballerina runtime won't be backward compatible with jdk8, so thought of limiting the debugger compatibility only to jdk11. @keizer619 any thoughts?

Comment (keizer619) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/utils/DebugUtils.java:14]: Yes runtime will not support for 1.8

**Review by keizer619 — APPROVED** [review]

**Pull request merged**